modules/imgproc/test/test_color.cpp

```diff
@@ -164,20 +161,14 @@ void CV_ColorCvtBaseTest::run_func()
     CvArr* out0 = test_array[OUTPUT][0];
     cv::Mat _out0 = cv::cvarrToMat(out0), _out1 = cv::cvarrToMat(test_array[OUTPUT][1]);
 
-    if(!test_cpp)
-        cvCvtColor( inplace ? out0 : test_array[INPUT][0], out0, fwd_code );
-    else
-        cv::cvtColor( cv::cvarrToMat(inplace ? out0 : test_array[INPUT][0]), _out0, fwd_code, _out0.channels());
+    cv::cvtColor( cv::cvarrToMat(inplace ? out0 : test_array[INPUT][0]), _out0, fwd_code, _out0.channels());
```

Comment:
`cvarrToMat` is legacy call. 5.x API should not provide `cvArr` type.

Reply:
This will be revisited later, during _core_ module cleanup.

I'm afraid it would be too hard to remove `cvarrToMat` completely as it is being used extensively in many places (~400 calls), e.g. _videoio_ backends. All these parts would need to be rewritten from `IplImage` to `cv::Mat`. As the first step we can "hide" part of C-API from the user (move to private headers).